src/main/kotlin/no/nav/familie/ef/iverksett/iverksetting/tilstand/TilstandRepository.kt

```diff
@@ -55,10 +55,15 @@ class TilstandRepository(val namedParameterJdbcTemplate: NamedParameterJdbcTempl
     }
 
 
-    fun oppdaterJournalpostResultat(behandlingId: UUID, journalPostResultat: JournalpostResultat) {
+    @Transactional(propagation = Propagation.REQUIRES_NEW)
```

Comment:
Nå er ikke jeg noen transaksjonsgud, men mener å huske at transaksjon mellom lagring/uthenting er pain;
For 2 mottakere skjer dette:
Transaksjon A starter tasken
Transaksjon B lagrer ned mottaker1
Transaksjon C henter ut resultat (mottaker1) og lagrer ned mottaker1 og mottaker2

Med oppsettet vi har nå - finner `Transaksjon C` faktisk `mottaker1`? 😂

Reply:
Hm, det ser poenget ditt. Men ville ikke integrasjonstesten `skal oppdatere journalpostresultat for brevmottakere` feilet hvis det var et issue?

Reply:
Jeg skal sjekke :)